newsfragments/validate_files.py

```diff
@@ -5,12 +5,12 @@
 
 import os
 import pathlib
+import sys
 
 ALLOWED_EXTENSIONS = {
-    '.feature.rst',
     '.bugfix.rst',
     '.doc.rst',
-    '.removal.rst',
```

Comment:
Maybe this is meant to be for backwards incompatible changes? It probably does make sense to have its own category for that, but we don't need to figure it out right now. :+1: for leaving it out.

Reply:
Ah, didn't even realize `removal.rst` was in the existing extentions - thought that came over from Trinity. Wasn't trying to make a statement there, so I'll add it back in.